SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -45,6 +45,7 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, WorkerLogLevel(ESettingsWorkerLogVerbosity::Warning)
 	, bEnableUnrealLoadBalancer(false)
 	, bUseRPCRingBuffers(false)
+	, bRunSpatialWorkerConnectionOnGameThread(false)
```

Comment:
Can you add a command line override for this setting also (within USpatialGDKSettings::PostInitProperties)

Reply:
Added.